pkg/test/framework/components/registryredirector/registryredirector.go

```diff
@@ -12,43 +12,48 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-// Package containerregistry provides basic utilities around configuring the fake
-// container registry server component for integration testing.
-package containerregistry
+// Package registryredirector provides basic utilities around configuring the fake
+// image registry server component for integration testing.
+package registryredirector
 
 import (
 	"istio.io/istio/pkg/test"
 	"istio.io/istio/pkg/test/framework/components/cluster"
 	"istio.io/istio/pkg/test/framework/resource"
 )
 
-// Instance represents a deployed fake container registry app instance.
+// Instance represents a deployed registry redirector app instance.
 type Instance interface {
 	// Address is the address of the service provided by the
-	// fake container registry server.
+	// registry redirector server.
 	Address() string
 
-	// SetupTagMap posts the tag map to the faked containerregistry.
+	// SetupTagMap posts the tag map to the registryredirector.
 	SetupTagMap(map[string]string) error
 }
 
-// Config defines the options for creating an fake container registry component.
+// Config defines the options for creating an registry redirector component.
 type Config struct {
 	// Cluster to be used in a multicluster environment
 	Cluster cluster.Cluster
+	// Upstream registry. Default is "gcr.io".
+	TargetRegistry string
+	// Docker image location of the fake registry. Default is "gcr.io/istio-testing/fake-registry:x.x".
+	// Please refer to registry_redirector_server.yaml for the exact default image.
+	RegistryRedirectorImage string
```

Comment:
nit: i'd just call this something simpler like `Image`.  this is already inside of `registryredirector.Config`.

Reply:
Changed to `Image`.